packages/client-app/src/components/AddressBar/AddressBarServer.vue

```diff
@@ -36,6 +36,20 @@ const updateSelectedServer = (serverUid: string) => {
 const isSelectedServer = (serverId: string) => {
   return activeCollection.value?.selectedServerUid === serverId
 }
+
+// Replace variables in URL
```

Comment:
For variable definitions and functions the `/** Comment */` format is better because it shows up in intellisense. Rather than labelling the computed property with whats going on inside its better to label the variable with what it represents